wordpress-mobile/WordPress-Android
# View all Stats

Fixes #8815.

This PR implements the View all screens for individual stats types and it brings the new stats refresh functionality on par with the old implementation.

This is just an aggregate PR of already reviewed partial ones.

To test:

1. Go to Stats
2. Wait for data to load
3. Tap on the View more button for different stats types
4. Observe the expected data is loaded

Update release notes:
